sale_delivery_date/models/res_partner.py

```diff
@@ -47,6 +51,7 @@ def next_delivery_window_start_datetime(self, from_date=None, timedelta_days=Non
         if timedelta_days is None:
             timedelta_days = 7
```

Comment:
7 is not mutable. Why don't you put this in the signature of the method?

Reply:
Actually it's there already.